indexclient/client.py

```diff
@@ -40,13 +38,17 @@ def check_status(self):
     def get(self, did):
         """Return a document object corresponding to a single did"""
         try:
-            self._get("index", did)
+            response = self._get("index", did)
         except requests.HTTPError as e:
             if e.response.status_code == 404:
                 return None
             else:
-                raise e
-        return Document(self, did)
+                print("Exception!!" + str(e.reason))
```

Comment:
I think it's cleaner to not overload `None` to mean both not found and something wrong happened?